language/java/run-tests.md

```diff
@@ -149,13 +149,16 @@ $ docker build -t java-docker --target test .
 
 The build output is truncated for simplicity, but you can see that our tests ran succesfully and passed. Let’s break one of the tests and observe the output when our tests fail.
 
-Open the `src/test/java/org/springframework/samples/petclinic/model/ValidatorTests.java` file and change **line 57** to the following.
+Open the `src/test/java/org/springframework/samples/petclinic/model/ValidatorTests.java` file and change the assertion 
 
 ```shell
```

Comment:
this should probably use `java` as code-hint now (and the leading whitespace removed) 🤔